### PR TITLE
Add demo data

### PR DESCRIPTION
**Pull request by bwrsandman**
- Brussels Belgium city

In some modules which make reference to this module, I will be trying to add some demo data.
Having one demo better zip helps make that possible.

### CODE REVIEW

**Comment by bwrsandman** [comment]
If there are more demo datas which would be nice to put, please provide me some examples and I'll gladly include them.

**Comment by pedrobaeza** [comment]
`demo/better_zip.xml` file not included

**Comment by bwrsandman** [comment]
Thanks. I went too fast.

File: base_location/demo/better_zip.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version = "1.0" encoding="utf-8"?>
+<openerp>
+  <data>
+
+    <record id="demo_brussels" model="res.better.zip">
+      <field name="city">Brussels</field>
```
Comment (pedrobaeza) [inline base_location/demo/better_zip.xml:7]: No zip code?

Comment (bwrsandman) [inline base_location/demo/better_zip.xml:7]: Using it for City management rather than ZIP codes. I don't know any Belgian ZIP codes but I can add one.

Comment (pedrobaeza) [inline base_location/demo/better_zip.xml:7]: Well, it's for providing a full example record.

**Comment by bwrsandman** [comment]
I assume 1000 is the zip code for Brussels.

**Comment by yvaucher** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**